command/root.go

```diff
@@ -3,11 +3,14 @@ package command
 var Command RootCommand
 
 type RootCommand struct {
-	Version func()       `short:"v" long:"version" description:"Print the version and exit."`
-	List    ListCommand  `command:"list" alias:"ls" description:"List managed instances."`
-	Shell   ShellCommand `command:"shell" alias:"sh" description:"Start an interactive shell."`
-	Run     RunCommand   `command:"run" description:"Run a command on the targeted instances."`
-	AwsOpts AwsOptions   `group:"AWS Options"`
+	Version          func()                    `short:"v" long:"version" description:"Print the version and exit."`
+	List             ListCommand               `command:"list" alias:"ls" description:"List managed instances."`
+	Shell            ShellCommand              `command:"shell" alias:"sh" description:"Start an interactive shell."`
+	Run              RunCommand                `command:"run" description:"Run a command on the targeted instances."`
+	ListDocuments    ListDocumentsCommand      `command:"list-documents" alias:"ld" description:"List available documents in ssm."`
```

Comment:
Perhaps we need to consider using subcommands? E.g.:

- `run command` and `run document`?
- `list instances` and `list documents`?

Personally, I like it the way it is though 👍

Reply:
Sounds nice - I slightly prefer the `verb object` pattern for cli too, especially when there are more than 1 types of object on which the cli work 👍

Reply:
I kind of like the verb object pattern too. Shall we go for this?

Reply:
I'm all for it! Perhaps as a separate PR since it would be a 'breaking change' for the existing commands?

Reply:
Sounds good, I have a few tests I need to add but should be done with this PR pretty soon. Will do another PR for the `verb object` refactor.